modules/apps/headless/headless-batch-engine/headless-batch-engine-impl/rest-openapi.yaml

```diff
@@ -0,0 +1,183 @@
+                  name: callbackURL
+                  schema:
+                      type: string
+            requestBody:
+                content:
+                    multipart/form-data:
+                required: true
+            responses:
+                202:
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/ImportTask"
+                        application/xml:
+                            schema:
+                                $ref: "#/components/schemas/ImportTask"
+                    description: Async
+            tags: ["Task"]
+        get:
+            operationId: getImportTask
+            parameters:
+                - in: path
+                  name: taskId
+                  required: true
+                  schema:
+                      format: int64
+                      type: integer
+            responses:
+                202:
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/ImportTask"
+                        application/xml:
+                            schema:
+                                $ref: "#/components/schemas/ImportTask"
+                    description: Async
+            tags: ["ImportTask"]
+        post:
+            description: Uploads a new file for creating new items in batch.
+            operationId: postImportTask
+            parameters:
+                - in: path
+                  name: className
+                  required: true
+                  schema:
+                      type: string
+                - in: path
+                  name: version
+                  required: true
+                  schema:
+                      type: string
+                - in: query
+                  name: callbackURL
+                  schema:
+                      type: string
+            requestBody:
+                content:
+                    multipart/form-data:
+                        schema:
+                            properties:
+                                file:
+                                    format: binary
+                                    type: string
+                required: true
+            responses:
+                202:
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/ImportTask"
+                        application/xml:
+                            schema:
+                                $ref: "#/components/schemas/ImportTask"
+                    description: Async
+            tags: ["ImportTask"]
+        put:
+            description: Uploads a new file for updating items in batch.
+            operationId: putImportTask
+            parameters:
+                - in: path
```

Comment:
same as before, missing importTaskId (is it needed? if it isn't, just defined under the url /import-task/) and the path doesn't reflect className nor path, are those query params instead of path params? if they are indeed path params, you should reflect it in the path, like /import-task/className/version/